types/part_set.go

```diff
@@ -517,10 +529,10 @@ func (ps *PartSet) AddPart(part *Part) (bool, error) {
 		return false, fmt.Errorf("%w:%w", ErrPartSetInvalidProofHash, err)
 	}
 
-	return ps.AddPartWithoutProof(part)
+	return ps.addPart(part)
 }
 
-func (ps *PartSet) AddPartWithoutProof(part *Part) (bool, error) {
+func (ps *PartSet) addPart(part *Part) (bool, error) {
```

Comment:
celestia node and I think other things rely on this so we need to keep this for now